src/main/java/io/smartcat/kinesis/cassandra/DefaultCassandraEmitter.java

```diff
@@ -47,15 +48,15 @@ public void init(CassandraKinesisConnectorConfiguration config) {
         if (config == null) {
             throw new IllegalStateException("Emitter has to be initialized first.");
         }
-        List<String> inserts = prepareInserts(buffer.getRecords());
-        executeInsertsAsync(inserts);
+        insert(buffer.getRecords());
+
         return new ArrayList<>();
```

Comment:
@nivancevic Why do we return new array list here? Should we return something meaningful or its just interface enforced?